### ISSUE TITLE
Extension is incompatible with VS 17.4 preview 1.0

### ISSUE THREAD

**Issue by PlonkaAdam** [microsoft/TemplateStudio#4589]
### Describe the bug

Missing method error appears after trying to use the extension in VS 17.4

```
System.MissingMethodException: Method not found: 'Void Microsoft.TemplateEngine.Edge.DefaultTemplateEngineHost..ctor(System.String, System.String, System.Collections.Generic.Dictionary`2<System.String,System.String>, System.Collections.Generic.IReadOnlyList`1<System.ValueTuple`2<System.Type,Microsoft.TemplateEngine.Abstractions.IIdentifiedComponent>>, System.Collections.Generic.IReadOnlyList`1<System.String>, Microsoft.Extensions.Logging.ILoggerFactory)'.
   at Microsoft.Templates.Core.CodeGen.CreateHost(String locationId, String hostVersion)
   at Microsoft.Templates.Core.CodeGen..ctor(String locationId, String hostVersion)
   at Microsoft.Templates.Core.CodeGen.Initialize(String locationId, String hostVersion, String templatePath)
   at Microsoft.Templates.Core.Gen.GenContext.Bootstrap(TemplatesSource source, IGenShell shell, String wizardVersion, String platform, String language)
   at Microsoft.Templates.UI.VisualStudio.SolutionWizard.Initialize()
   at Microsoft.Templates.UI.VisualStudio.SolutionWizard.RunStarted(Object automationObject, Dictionary`2 replacementsDictionary, WizardRunKind runKind, Object[] customParams)
   at Microsoft.VisualStudio.TemplateWizard.Wizard.Execute(Object application, IntPtr hwndOwner, Object[]& ContextParams, Object[]& CustomParams, wizardResult& retval)
```

### To Reproduce

 1. Install plugin
 2. Try to create new project

### Applies to the following platforms:

- [X] WinUI
- [ ] WPF
- [ ] UWP

### About your setup

 - Visual Studio Version:  17.4 preview 1
 - Template Studio Wizard Version:  5.2.1
 - Windows Build: 22000.795

**Comment by mikebattista** [comment]
Updating to https://www.nuget.org/packages/Microsoft.TemplateEngine.Edge/7.0.100-preview.7.22376.2 seems to resolve the issue in 17.4 but breaks 17.3. Sent mail to the package owners for guidance.

**Comment by mikebattista** [comment]
I've been told this will be fixed in 17.4 Preview 2.

**Comment by mikebattista** [comment]
This is fixed in 17.4 Preview 2.
